helpers/assert_soundcard_present

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+. bootrr
+
+TEST_CASE_ID="$1"
+SOUNDCARD="$2"
+DEVICE="$3"
+TIMEOUT="${4:-1}"
+
+if [ -z "${TEST_CASE_ID}" -o -z "${SOUNDCARD}" -o -z "${DEVICE}" ]; then
+	echo "Usage: $0 <test-case-id> <soundcard> <device> [<timeout>]"
+	exit 1
+fi
+
+timeout ${TIMEOUT} [ -d /proc/asound/${SOUNDCARD} ] || test_report_exit blocked
```

Comment:
why blocked here? if the soundcard is missing, it should be fail, no?

Reply:
@ndechesne Right, I will change to failure state.